### DISCUSSION TITLE
2023 version 2

### DISCUSSION THREAD

**Discussion by edupont04** [microsoft/dynamics365patternspractices#6] [Announcements]
April 2023: Updated templates, updated business process catalog. Learn more at https://learn.microsoft.com/en-us/dynamics365/guidance/

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/dynamics365patternspractices/releases/tag/2023.2'>2023 version 2</a>.</em>

**Comment by ClaraSeijoBlanque** [comment]
Hi Eva,

I was already using the previous version (JAN 23), it's really useful. Reviewing the new file I'm missing the correct order for the processes because in the new version the columns "End-to-end business process", "Business process area" and "Business process"   are sort by name, but this is not the correct sort.

Example, for _Acquire to dispose --> Acquire assets_, the first related process in the excel is "Accumulate acquisition cost" and this is not the real order.

Could you please upload the file with the process order?

Thanks in advance
Clara

**Reply by edupont04** [reply]
Hi Clara,
We're delighted to hear that you find the catalog useful! But I'm not sure what you mean with the sort order. If you download the Excel book, you can sort the entries any way you like, but the default sort order says this:
End-to-end business process	Business process area	Business process
Acquire to dispose	                Acquire assets	                Accumulate acquisition cost
Acquire to dispose         	       Acquire assets	                Accumulate acquisition costs
Acquire to dispose      	      Acquire assets	               Accumulate acquisition costs
Acquire to dispose      	      Acquire assets      	       Classify asset leases

**Comment by ClaraSeijoBlanque** [comment]
Hi Eva,

Thanks for your answer! Yes, being an excel I understand I can sort as I like, but what I'm missing is the correct sequence in running the processes. For example, let me change the area to understand it easier; for **Procure to pay > Manage vendor relationships** in the new excel the first process is "_Add vendor bank accounts_" but the sequence order is: 

![2023-04-18_15-50-30](https://user-images.githubusercontent.com/131162800/232799064-abb0dae8-367f-4f6a-b007-7e5a4e80c90f.jpg)

That mean, I can create the vendor bank account but only if the vendor is created. So, if I need to follow the process in a flow or in the system I need to do it using this sequence. This order was included in the previous version without number but with the correct sort (without being sorted by name). This is the new version:

![2023-04-18_15-57-14](https://user-images.githubusercontent.com/131162800/232800382-ccbdb220-3561-4b06-881c-00c58814e599.jpg)

And this was the old one for the same processes area:

![2023-04-18_15-58-27](https://user-images.githubusercontent.com/131162800/232800731-3209579b-4fea-495f-ab38-80de4f83a288.jpg)

I hope my question is more clear now.

Thanks again!
Clara

**Reply by edupont04** [reply]
Thanks for the clarification! The new version is sorted alphabetically, not in order of sequence for the individual business processes. It also doesn't reflect the order in which we'll eventually publish the various articles at https://learn.microsoft.com/en-us/dynamics365/guidance/business-processes/overview
Adding @rachel-profitt for potential insights, since she manages the Excel workbook.

**Reply by rachel-profitt** [reply]
We are excited to announce that the latest version of the catalog now has a sequence ID added to the catalog.
